osi_environment.proto

```diff
@@ -31,7 +31,7 @@ message EnvironmentalConditions
     // is Unix time 0 (midnight 1/1/1970), but 'epoch' is often used as
```

Comment:
According to CCB: A short comment that the unix timestamp refers to the ego vehicle (and for example not to a time stamp of a simulator) should be added. Maybe "The Unix timestamp of the host vehicle."
@thomassedlmayer Could you add a short comment? If this leads to further discussion we would handle this in a separate PR then.